test/functional/apps/dashboard_elements/controls/options_list.ts

```diff
@@ -392,56 +388,139 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
-            await dashboardControls.optionsListPopoverSelectOption('exists');
-            await dashboardControls.optionsListEnsurePopoverIsClosed(controlId);
-            await dashboard.waitForRenderComplete();
+        before(async () => {
+          await common.navigateToApp('settings');
+          await settings.clickKibanaIndexPatterns();
+          await settings.clickIndexPatternByName('animals-*');
+          await settings.addRuntimeField(
+            FIELD_NAME,
+            'keyword',
+            `emit(doc['sound.keyword'].value.substring(0, 1).toUpperCase())`
+          );
+          await header.waitUntilLoadingHasFinished();
 
-            expect(await pieChart.getPieSliceCount()).to.be(5);
-            expect((await pieChart.getPieChartValues())[0]).to.be(5);
+          await returnToDashboard();
+          await dashboardControls.deleteAllControls();
+        });
+
+        it('can create options list control on runtime field', async () => {
+          await dashboardControls.createControl({
+            controlType: OPTIONS_LIST_CONTROL,
+            fieldName: FIELD_NAME,
+            dataViewTitle: 'animals-*',
           });
+          expect(await dashboardControls.getControlsCount()).to.be(1);
+        });
 
-          it('negating exists query has expected results', async () => {
-            await dashboardControls.optionsListOpenPopover(controlId);
-            await dashboardControls.optionsListPopoverSetIncludeSelections(false);
-            await dashboardControls.optionsListEnsurePopoverIsClosed(controlId);
-            await dashboard.waitForRenderComplete();
+        it('new control has expected suggestions', async () => {
+          controlId = (await dashboardControls.getAllControlIds())[0];
+          await ensureAvailableOptionsEql(FIELD_VALUES);
+        });
 
-            expect(await pieChart.getPieSliceCount()).to.be(1);
-            expect((await pieChart.getPieChartValues())[0]).to.be(1);
-          });
+        it('making selection has expected results', async () => {
+          await dashboardControls.optionsListOpenPopover(controlId);
+          await dashboardControls.optionsListPopoverSelectOption('B');
+          await dashboardControls.optionsListEnsurePopoverIsClosed(controlId);
+          expect(await pieChart.getPieChartLabels()).to.eql(['bark', 'bow ow ow']);
         });
 
         after(async () => {
           await dashboardControls.deleteAllControls();
+          await dashboard.clickQuickSave();
+          await header.waitUntilLoadingHasFinished();
+
+          await common.navigateToApp('settings');
+          await settings.clickKibanaIndexPatterns();
+          await settings.clickIndexPatternByName('animals-*');
+          await settings.filterField('testRuntimeField');
+          await testSubjects.click('deleteField');
+          await settings.confirmDelete();
+        });
+      });
+
+      describe('test exists query', async () => {
```

Comment:
Since the `runtime field` tests also require adding data (but in a slightly different way), I decided to encapsulate the relevant data addition in to the respective `runtime field` and `exists query` test suites.